.node-version

```diff
@@ -0,0 +1 @@
+.nvmrc
```

Comment:
…is this just what a symlink looks like in git?

Reply:
Yeah 😅